apps/clinicapp/clinicapp.yml

```diff
@@ -0,0 +1,7 @@
+name: clinicapp
+description: 'Clinic Management App.'
+website: 'https://github.com/babitadelfino/clinicapp.git'
```

Comment:
Your need to replace `website` field with your official website page.

Reply:
Replaced link, please review